.changeset/modern-turkeys-hang.md

```diff
@@ -0,0 +1,5 @@
+---
+"@directus/app": patch
+---
+
+Fix saving/updating in Kanban-Layout with missing Sort-Field
```

Comment:
```suggestion
Fixed saving/updating in Kanban-Layout with missing Sort-Field
```

Reply:
Ahhhh the famous changeset typo happened to me now too, nooooo :smile: